src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/Dependencies/CrossTarget/AggregateCrossTargetProjectContextProvider.cs

```diff
@@ -86,7 +86,7 @@ public async Task<AggregateCrossTargetProjectContext> CreateProjectContextAsync(
         {
             if (string.IsNullOrEmpty(shortOrFullName))
             {
-                object? targetObject = await configurationGeneralProperties.TargetFramework.GetValueAsync();
+                object? targetObject = await configurationGeneralProperties.TargetFrameworkMoniker.GetValueAsync();
```

Comment:
I think this is wrong in the case of `net5.0-windows`; this results in:

```
TargetFrameworkMoniker; .NETCoreApp, Version=v5.0
TargetPlatformMoniker; Windows, Version=7.0
```

Either we represent both of these, or we need to continue to read this value. This really should be coming from the "TargetFramework" dimension from the ProjectConfiguration of the current project.

Reply:
I had started out expecting the "TargetFramework" dimension in configuredProject.ProjectConfiguration.Dimensions to be non-empty but that isn't the case. I thought that was by design. Let me dig into why that's happening.

Reply:
That is by design in non-multi-TFM projects. Maybe we need to sync up on this?